### PR TITLE
Add obsolesence QC check

### PR DESCRIPTION
**Pull request by matentzn**
close https://github.com/monarch-initiative/mondo/issues/5778

### CODE REVIEW

File: src/ontology/mondo.Makefile [hunk 1]
```diff
@@ -681,6 +681,15 @@ test_owlaxioms:
 
 test: test_owlaxioms 
 
+.PHONY: test_obs_reason
+test_obs_reason:
+	echo "all obsolesence reasons should be typed as xsd:string"
+	! grep -E "property_value: IAO:0000231.*xsd:[^s]" mondo-edit.obo
+	echo "obsolesence reason does not seem to use the correct property"
+	! grep -E "\"terms merged\" xsd:" mondo-edit.obo | grep -v IAO:0000231
```
Comment (matentzn) [inline src/ontology/mondo.Makefile:689]: @nicolevasilevsky note this check only covers the "terms merged" case, and does not check other obsolescence reasons. You can copy paste this line to capture other obsolescence reasons.

Comment (nicolevasilevsky) [inline src/ontology/mondo.Makefile:689]:
```suggestion
	! grep -E "\"terms merged\" xsd:" mondo-edit.obo | grep -v IAO:0000231
	! grep -E "\"out of scope\" xsd:" mondo-edit.obo | grep -v IAO:0000231
	! grep -E "\"termssplit\" xsd:" mondo-edit.obo | grep -v IAO:0000231
```

Comment (matentzn) [reply src/ontology/mondo.Makefile:689]: termssplit one word? if so its fine.

Comment (nicolevasilevsky) [reply src/ontology/mondo.Makefile:689]: ha, no, that was a typo, thanks for catching that! I updated it

Comment (nicolevasilevsky) [reply src/ontology/mondo.Makefile:689]: updated

**Pull request merged**